### PR TITLE
Klendathu Arachnids

### PR DESCRIPTION
**Pull request by Viciousarus**
First pull request, let's see how I will fuck up

Sprites and code for Klendathu warrior bugs from Starship Troopers.

### CODE REVIEW

**Comment by DrSnips** [comment]
Why should these exist other than use having more EPIC GAMER references?

**Comment by tuypo1** [comment]
The server is called /vg/ for videogame generals, epic gamer references are a good thing, I'm not sure if there has ever been a starship troopers vidya though

Still can't really complain about more mob types, dumbass should have provided a picture of the spirits though

**Comment by Blithering** [comment]
More alien mobs that are actually aliens is nice.

**Comment by strawberrytan** [comment]
You can view the sprites by going to commits > sprites, any image viewer can open a dmi as if it was a long and on mobile devices you can open it using vlc

**Comment by Exxion** [comment]
You can also click the details button on the Icon Diffs check below

**Comment by tuypo1** [comment]
Ok saw the spirits, if they don't have wings why can they pass over tables?

**Comment by Exxion** [comment]
Same way dwarves can, presumably

**Comment by SonixApache** [comment]
I WANT TO KNOW MORE

**Comment by Viciousarus** [comment]
They can no longer casually walk over tables, but if they see something tasty they can just bash through them no problem.

**Comment by tuypo1** [comment]
Much better

File: code/modules/mob/living/simple_animal/hostile/warriorbug.dm [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	icon_living = "arachnid"
+	icon_dead = "arachnid_dead"
+	speak_emote = list("roars")
+	emote_hear = list("hisses")
+	speak_chance = 5
+	turns_per_move = 5
+	see_in_dark = 10
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/spidermeat
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
```
Comment (jknpj) [inline code/modules/mob/living/simple_animal/hostile/warriorbug.dm:15]: :thinking:

**Pull request merged**